examples/sand_and_gravel_marine_dredged/v2.json

```diff
@@ -0,0 +1,19 @@
+{
+  "schema_version": "v2",
+  "survey_id": "066",
```

Comment:
Something minor, but should the survey_id here be "076"?

Reply:
Fixed